src/Device/DeviceSelector/DeviceSelector.tsx

```diff
@@ -62,6 +64,17 @@ export default ({
     const showSelectedDevice = deviceIsSelected || waitingToAutoReconnect;
 
     const doDeselectDevice = useCallback(() => {
+        dispatch(
+            (): AppThunk => (_, getState) => {
+                const device = getState().device.selectedDevice;
+                if (device) {
+                    usageData.sendUsageData(
+                        'device deselected ',
+                        simplifyDeviceForLogging(device)
+                    );
+                }
+            }
+        );
```

Comment:
```suggestion
    const doDeselectDevice = useCallback((device: Device) => {
          usageData.sendUsageData(
              'device deselected ',
              simplifyDeviceForLogging(device)
          );
```

Alternatively, you can have it as an optional parameter to avoid the "currentDevice is of type Device | undefined" when calling it further down the file.